zio-cli/shared/src/main/scala/zio/cli/PrimType.scala

```diff
@@ -327,7 +327,7 @@ object PrimType {
     def validate(value: Option[String], conf: CliConfig): IO[String, JYearMonth] = {
       val AcceptedFormat = "^(-?\\d+)-(\\d{2})".r
       def parse(input: String) = input match {
-        case AcceptedFormat(y, m) => ZIO(JYearMonth.of(y.toInt, m.toInt))
+        case AcceptedFormat(y, m) => ZIO.succeed(JYearMonth.of(y.toInt, m.toInt))
```

Comment:
Can this throw exceptions? In prior ZIO versions `ZIO.apply` was equivalent to `ZIO.attempt` and `UIO.apply` was equivalent to `UIO.succeed` so `attempt` would actually be the more faithful translation here.

Reply:
Done